inc/user.class.php

```diff
@@ -3799,6 +3799,7 @@ static function getSqlSearchResult ($count = true, $right = "all", $entity_restr
                            'glpi_profilerights.rights'   => ['&', KnowbaseItem::READFAQ]
                         ] + getEntitiesRestrictCriteria('glpi_profiles_users', '', $entity_restrict, 1)
                      ];
+                     break;
```

Comment:
According description of #561 :roll_eyes: , I guess the lack of `break` was just a mistake. So OK for this one.